test/src/e2e_vm_tests/test_programs/should_pass/forc/dependency_patching/.gitignore

```diff
@@ -0,0 +1,2 @@
+out
+target
```

Comment:
I think these should be covered by the top-level `.gitignore`?

Reply:
yep, couldn't decide what was our general approach here while adding tests under e2e, deleting it 👍